src/insns/condbr_common.adoc

```diff
@@ -1,5 +1,5 @@
 Exceptions::
 When the target address is not within the <<pcc>>'s bounds, and the branch is taken,
 a _CHERI jump or
-branch fault_ is reported in the TYPE field and Length Violation is reported in
```

Comment:
I assume the capital V was previously rogue and there are no other instances of that?

Reply:
It was a typo. I found and fixed another instance of this in the mret/sret instruction page.

Reply:
seems to be fixed